tests/phpunit/tests/editor/enableContentEditorForNavigationPostType.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @group editor
+ *
+ * @covers ::_enable_content_editor_for_navigation_post_type
+ */
+class Tests_Editor_EnableContentEditorForNavigationPostType extends WP_UnitTestCase {
+
+	const NAVIGATION_POST_TYPE     = 'wp_navigation';
+	const NON_NAVIGATION_POST_TYPE = 'wp_non_navigation';
+
+	public function tear_down() {
+		$this->enable_editor_support();
+		parent::tear_down();
+	}
+
+	/**
+	 * @ticket 56266
+	 *
+	 * @dataProvider data_test_should_only_enable_content_editor_for_navigation_post_types
```

Comment:
```suggestion
	 * @dataProvider data_should_only_enable_content_editor_for_navigation_post_types
```

Reply:
Fixed in d6ee08b6441d289e4009cbf66b298f3959fa54ad